pychunkedgraph/graph/client/bigtable/utils.py

```diff
@@ -71,12 +73,35 @@ def _get_column_filter(
     )
 
 
+def _get_user_filter(user_id: str):
+    """generates a ColumnRegEx Filter which filters user ids
+
+    Args:
+        user_id (str): userID to select for
+    """
+
+    condition = RowFilterChain(
+        [
+            ColumnQualifierRegexFilter(attributes.OperationLogs.UserID.key),
+            ValueRangeFilter(str.encode(user_id), str.encode(user_id)),
+            CellsRowLimitFilter(1),
```

Comment:
What is the additional `CellsRowLimitFilter` for?

Reply:
I'm not sure I totally understand to be honest... maybe it would work without it, but I think the other filter requires a simple True/False to work correctly, so if there are multiple columns which include a userID perhaps this would return a list rather than 1 per row?  Maybe a theoretical concern that isn't a problem for our data?